pre-edited-cs/data/Stage/ScriptSource/Almond.txt

```diff
@@ -117,12 +112,11 @@ Don't you think?<NOD<END
 #0306
 <PRI<MSG<TURThere is no reply...<NOD<ITJ0030:0307<END
 #0307
-<PRI<SAT<MSGShe isn't moving, but it looks
-like she's still alive.<NOD<CLR
+<PRI<SAT<MSG<TUR
```

Comment:
<TUR isn't necessary if there's a <SAT. <SAT keeps working until the next <END or <TRA (including across <EVEs or other event jumps)